### PR TITLE
Add note to generate-uuid docs

### PR DESCRIPTION
**Pull request by greeshmaswaminathan**
Add note to generate-uuid docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ These directives are currently available:
 | [Decode](wrangler-docs/directives/decode.md)                                    | Decodes a column value as one of `base32`, `base64`, or `hex`    |
 | [Encode](wrangler-docs/directives/encode.md)                                    | Encodes a column value as one of `base32`, `base64`, or `hex`    |
 | **Unique ID**                                                          |                                                                  |
-| [UUID Generation](wrangler-docs/directives/generate-uuid.md)                    | Generates a universally unique identifier (UUID)                 |
+| [UUID Generation](wrangler-docs/directives/generate-uuid.md)                    | Generates a universally unique identifier (UUID) .Recommended to use with CDAP version 6.4 and above due to an important bug fix [CDAP-17732](https://cdap.atlassian.net/browse/CDAP-17732)             |
```
Comment (albertshau) [inline README.md]: should mention the wrangler version, not the CDAP version.

Comment (greeshmaswaminathan) [reply README.md]: changed.

**Review by albertshau — APPROVED** [review]

**Pull request merged**